### DISCUSSION TITLE
Theme for VSCode

### DISCUSSION THREAD

**Discussion by LyiarOwl** [diegoulloao/neofusion.nvim#10] [Ideas]
Hii, please create a version for VSCode 🥹

**Comment by diegoulloao** [comment]
> Hii, please create a version for VSCode 🥹

https://github.com/diegoulloao/neofusion.vscode/
Available at Marketplace 🙂

**Reply by LyiarOwl** [reply]
woooooo... 😱 i need to kiss your mouth ❤️‍🔥

i was all this time using a theme similar to neofusion but i really wanted the original neofusion 🥰😍😍

**Reply by diegoulloao** [reply]
hey, glad you like it 😂 ❤️ 
please feel free to open any issue if you find one 🙂
